### DISCUSSION TITLE
Possible ROFI logo?

### DISCUSSION THREAD

**Discussion by isidyaG** [davatorium/rofi#1816] [Theming]
Hello everyone,

Recently, I've started using rofi, as well theming it to my own needs. As I was searching through "Qball's Weblog",
I came across a post from the archive, made in 24.11.2014, that says rofi needs a logo.  So, I thought, why not? Let's try...

Here below you can see the PNG, it's something that I've came up for now, but nevertheless, 
it can be tailored to fit better in need. It can be made simpler, less colorful, or just typography logo! 

I hope you like it! ^..^

NOTE: It's voluntary work. :) 

![rofifinal](https://user-images.githubusercontent.com/126459733/221551693-b523bb0c-e4ef-43a8-9f20-5a609673792e.png)

**Comment by DaveDavenport** [comment]
Looks pretty nice.  Lot better then the current "logo"

![rofi](https://github.com/davatorium/rofi/blob/next/data/rofi.png?raw=true)

**Reply by isidyaG** [reply]
:D thank you! I was also thinking to make it simpler, it's all up to you, how you would prefer. 

I'm sorry if I posted this under wrong post, was thinking where exactly to put it. :)

But yes, if you would like some modifications, feel free to contact me, otherwise, feel free to use it! ^..^

**Reply by DaveDavenport** [reply]
If it would be possible to make a simpler version that could be used as icon, that would be great!
Then Ill include it as the application icon.

**Reply by isidyaG** [reply]
Hi there! I am sorry it took me some time to answer back, but was quite a busy period! 

What do you think about this logo? ^..^ I did make some other versions, but somehow this picked my eye :) 

![Rofi (1)](https://user-images.githubusercontent.com/126459733/231557615-563cd150-4a42-4df6-b57b-3bdba118f0b5.png)

If you like it, I can send it to you in any format you would need, also cropped to fit the content. :)

**Reply by DaveDavenport** [reply]
![image](https://user-images.githubusercontent.com/406245/231559596-d296c70b-d827-469d-94c0-a41caeef049f.png)
I like it. I think it would be nice if we can make an icon version (64x64/128x128) version (not sure how)

**Reply by isidyaG** [reply]
I just converted it from https://www.freeconvert.com/ : here is the output:

The first one is 64x64:
![rofi-64x64](https://user-images.githubusercontent.com/126459733/231563671-10dbdd83-343d-4eb3-8c6f-a7319cdc0a42.png)

The second one is 128x128:
![rofi-128x128](https://user-images.githubusercontent.com/126459733/231564079-b51f120c-2249-48fb-8d79-11eb50efcac8.png)

**Reply by isidyaG** [reply]
I have no idea why they look almost the same :D but they are different. The format is PNG, but it could be converted to ICO as well, I am not sure which one is better for implementing it :)

**Reply by DaveDavenport** [reply]
![image](https://user-images.githubusercontent.com/406245/231565175-fe993b6e-3a98-41fc-9c62-d187982f4cc4.png)
I think it did not scale..

**Reply by DaveDavenport** [reply]
It actually is still readable at 64x64 so that is good.  I was worried about that.
Normally you want to remove details for smaller icons..

F.e. this one that was made for gpx-viewer:

![image](https://user-images.githubusercontent.com/406245/231565734-7ef66a96-69b8-41cd-abca-8c2f91304190.png)

**Reply by isidyaG** [reply]
What do you mean "by not scale"? It's quite smaller :thinking:

**Reply by DaveDavenport** [reply]
here the two images you pasted are still pretty large (larger then stated size)
![image](https://user-images.githubusercontent.com/406245/231566230-ffcd67ab-ec69-406c-b6dc-be84be13a55f.png)
could be a github thing?

**Reply by isidyaG** [reply]
Weird, that was also my question before :D They seem large, but I did the 64x64 scale ..

**Reply by DaveDavenport** [reply]
![image](https://user-images.githubusercontent.com/406245/231569230-82950890-b8a5-45f6-80a8-2d67c5611a42.png)

**Reply by DaveDavenport** [reply]
yeah I think this will work as icon fine.

Thanks.

**Reply by isidyaG** [reply]
You're welcome! ^..^ me is happy :D
